### DISCUSSION TITLE
Switching wallets from Talisman EVM to PJS Substrate creating an Application Error loop

### DISCUSSION THREAD

**Discussion by polkadirk** [OAK-Foundation/OAK-blockchain#495] [Automation Hub]
If a user connects with the Talisman extension EVM wallet instead of Metamask (the user is auto-prompted to do this when connecting if they have the Talisman extension) and then attempts to reconnect to a substrate wallet it will cause an infinite application error loop. Video is attached below.

Disabling of the Talisman wallet extension provides a temporary fix to the issue.

https://github.com/OAK-Foundation/OAK-blockchain/assets/104081857/8ef7b95d-2f25-49a3-adb0-c3482c57c4a0

**Comment by v9n** [comment]
This should be fixed now @polkadirk
